be/src/exec/workgroup/scan_task_queue.h

```diff
@@ -44,8 +48,12 @@ struct YieldContext {
     YieldContext& operator=(YieldContext&&) = default;
 
     bool is_finished() const { return yield_point >= total_yield_point_cnt; }
-    void set_finished() { yield_point = total_yield_point_cnt = 0; }
+    void set_finished() {
+        yield_point = total_yield_point_cnt = 0;
```

Comment:
set  yield_point = 0 and total_yield_point_cnt = 0?

Reply:
It is guaranteed that after executing set_finished assert is_finished() == true